### PR TITLE
Updated init action to include removal of jackson libraries

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +49,15 @@ We prepared Dataproc [initialization action](https://cloud.google.com/dataproc/d
 that allows to run Oozie 5.1.0 on Dataproc.
 
 Please upload `dataproc/oozie-5.1.sh` to your GCS bucket and create cluster using following command:
+
+Note that you need at least 20GB RAM to run Oozie jobs on the cluster. The custom machine type before has enough RAM
```
Comment (sprzedwojski) [inline README.md]:
```suggestion
Note that you need at least 20GB RAM to run Oozie jobs on the cluster. The custom machine type below has enough RAM
```
I'm guessing you meant "below" - this way it makes more sense :)

File: README.md [hunk 2]
```diff
@@ -49,12 +49,15 @@ We prepared Dataproc [initialization action](https://cloud.google.com/dataproc/d
...
+to handle oozie.
```
Comment (sprzedwojski) [inline README.md]:
```suggestion
to handle Oozie.
```

**Comment by potiuk** [comment]
@sprzedwojski  - all fixed.

**Comment by potiuk** [comment]
@mik-laj @sprzedwojski - I am merging in a moment. I updated also commit description to include memory parameters setup that I added in the meantime.

**Pull request merged**